src/wasm/wasm-validator.cpp

```diff
@@ -947,10 +947,14 @@ void FunctionValidator::visitCall(Call* curr) {
 
 void FunctionValidator::visitCallIndirect(CallIndirect* curr) {
```

Comment:
Other table-accessing instructions like `table.get`, `table.set`, and the bulk table operations will need updating and testing as well.

Reply:
Yes, I mentioned that in the PR description this this is very must the first pass.  Hopefully we can import tests from upstream spec tests soon and I can use those?

I'm also way less interesting into those other instructions right now since LLVM doesn't generate them for the `__indirect_function_table`

Reply:
I did the rest!